scripts/globals/spells/maidens_virelai.lua

```diff
@@ -2,27 +2,52 @@
 -- Spell: Maiden's Virelai
 -- Charms pet
 -----------------------------------------
-
 require("scripts/globals/status");
+require("scripts/globals/magic");
+require("scripts/globals/pets");
 
 -----------------------------------------
 -- OnSpellCast
 -----------------------------------------
 
 function onMagicCastingCheck(caster,target,spell)
     -- Ix'Aern DRG pets are Wyverns that 2hour.
-    if (mob:getID() >= 16921023 and mob:getID() <= 16921025) then
-        if (caster:getStatusEffect(EFFECT_SOUL_VOICE)  == nil) then
+    if (caster:getID() >= 16921023 and caster:getID() <= 16921025) then
+        -- This would probably be better to handle in the mobs script but eh w/e..
```

Comment:
this would indeed be better handled in the mobs script! that's why you are going to remove it from here